### PR TITLE
Document signature verification for Telegraf binaries

### PR DESCRIPTION
**Pull request by pierwill**

### CODE REVIEW

**Comment by pierwill** [comment]
(Will probably revise as an ordered list—)

File: content/telegraf/v1.12/introduction/installation.md [hunk 1]
```diff
@@ -207,6 +207,35 @@ sudo systemctl start telegraf
 {{< /tab-content-container >}}
 {{< /tab-labels >}}
 
+### Verify the authenticity of downloaded binary
```
Comment (sanderson) [inline content/telegraf/v1.12/introduction/installation.md]: I think we can call out that this is an optional step, but recommended.

File: content/telegraf/v1.12/introduction/installation.md [hunk 2]
```diff
@@ -207,6 +207,35 @@ sudo systemctl start telegraf
...
+
+InfluxData cryptographically signs each Telegraf binary release.
+To verify the signature of your download, first download and import our public key:
```
Comment (sanderson) [inline content/telegraf/v1.12/introduction/installation.md]: We need to call out that users will need to install `gpg` and point to installation instructions for it.

**Comment by sanderson** [comment]
Whatever changes you make in `content/telegraf/v1.12/` need to be ported to `content/telegraf/v1.13` on the `telegraf-1.13` branch. There is currently a PR open for this branch.

**Pull request merged**